### PR TITLE
Yangg/frs whole summary

### PR DESCRIPTION
**Pull request by yangg-msft**
Update RouterliciousTestDriver to enable the whole summary when running tests against FRS.

### CODE REVIEW

File: packages/test/test-drivers/src/routerliciousTestDriver.ts [hunk 1]
```diff
@@ -57,7 +59,7 @@ function getEndpointConfigFromEnv(r11sEndpointName: string) {
     }
     assert(configStr, `Missing config for ${r11sEndpointName}`);
     const config = JSON.parse(configStr);
-    return getConfig(config.host, config.tenantId, config.tenantSecret);
+    return getConfig(config.host, config.tenantId, config.tenantSecret, config.useWholeSummary);
```
Comment (curtisman) [inline packages/test/test-drivers/src/routerliciousTestDriver.ts:62]: >useWholeSummary [](http://example.com/codeflow?start=79&length=15)

should we just allow the whole IRouterliciousDriverPolicies here?

Comment (tanviraumi) [reply packages/test/test-drivers/src/routerliciousTestDriver.ts:62]: How do we do that? Stringified JSON?

**Review by tanviraumi — APPROVED** [review]
Approving it as it is for now. We will follow up with @curtisman later

**Review by curtisman — APPROVED** [review]
:shipit:

**Comment by yangg-msft** [comment]
Closed accidentally.

**Pull request merged**